app/src/common/css/variables/z-index-variables.scss

```
@@ -26,6 +26,7 @@ $Z-INDEX-PAGE-CONTAINER-HEADER: 11;
 $Z-INDEX-HINT: 15;
 $Z-INDEX_AUTOCOMPLETE: 16;
 $Z-INDEX-OVERLAY-PANEL: 100;
-
+// ui-kit Modal above side panels (Modal uses this via MODAL_Z_INDEX in JS).
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)